e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -466,16 +470,19 @@ describe("scenarios > collection defaults", () => {
             cy.button("Move").click();
           });
 
-          cy.findByText("Orders").should("not.exist");
+          appContent().findByText("Orders").should("not.exist");
+          appContent().findByText("A nested collection").should("not.exist");
           cy.findByText(/item(s)? selected/).should("not.be.visible");
```

Comment:
My usage of `"not.exist"` vs `"not.be.visible"` is intentionally inconsistent. The `item(s) selected` (`BulkActions.jsx`) component is actually still mounted on the DOM but just moved off screen in order to create smooth hide/show animation, thus `"not.exist"` does not work. 

`"not.be.visible"` also doesn't work for the previous two lines (e.g. `cy.findByText("Orders").should("not.exist");`) because in those scenarios the component is actually removed from the DOM, so cypress requires us to use `"not.exist"` instead of `"not.be.visible"`.

If anyone has a suggestion on a better way to do this that is agnostic of whether the element is either removed from the DOM or just not visible on screen I would love to hear it 🙂